components/ad/smartbanner/src/index.scss

```diff
@@ -51,11 +51,15 @@
     }
   }
 
-  &-visible {
+  &--visible {
     margin-top: 70px;
   }
 
-  &-hidden {
+  &--hidden {
     margin-top: 0;
   }
```

Comment:
Well, another thing to discuss is... why are they called visible and hidden? But... "that's another war" 😄